dragonchain/lib/dto/bnb.py

```diff
@@ -0,0 +1,319 @@
+# Copyright 2019 Dragonchain, Inc.
+# Licensed under the Apache License, Version 2.0 (the "Apache License")
+# with the following modification; you may not use this file except in
+# compliance with the Apache License and the following modification to it:
+# Section 6. Trademarks. is deleted and replaced with:
+#      6. Trademarks. This License does not grant permission to use the trade
+#         names, trademarks, service marks, or product names of the Licensor
+#         and its affiliates, except as required to comply with Section 4(c) of
+#         the License and to reproduce the content of the NOTICE file.
+# You may obtain a copy of the Apache License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the Apache License with the above modification is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the Apache License for the specific
+# language governing permissions and limitations under the Apache License.
+
+import base64
+from typing import Dict, Any
+from ast import literal_eval
+
+from binance_chain.messages import TransferMsg, Signature
+from binance_chain.wallet import Wallet
+from binance_chain.environment import BinanceEnvironment
+import requests
+
+from dragonchain import logger
+from dragonchain import exceptions
+from dragonchain.lib.dto import model
+
+
+DC_MAINNET_NODE = "http://10.2.1.197"  # Mainnet BNB
+DC_TESTNET_NODE = "https://data-seed-pre-0-s3.binance.org:443/"  # Testnet BNB
+# we currently don't have a private testnet node set up...
+
+MAINNET_RPC_PORT = ":27147/"
```

Comment:
only require number, not part of a formatted string

Reply:
Fixed.